client/requirements.txt

```diff
@@ -1,4 +1,4 @@
-dragonfly >= 0.6.5
+dragonfly2 >= 0.8.0
```

Comment:
I haven't actually noticed that this file exists. I wasn't sure if anyone was using it, do you want to keep this file?